### PR TITLE
Add antitypical/Result as a dependency.

### PR DESCRIPTION
**Pull request by renatorodrigues**
This PR adds the Result framework to Alicerce as discussed in #123

### CODE REVIEW

**Comment by renatorodrigues** [comment]
I'm making Travis happy with this change, and indeed, we should update the README. Will do it afterward.

File: .travis.yml [hunk 1]
```diff
@@ -8,19 +8,27 @@ before_install:
...
+cache:
+  directories:
+    - $HOME/Library/Caches/org.carthage.CarthageKit/dependencies
+    - Carthage/Build
+
 jobs:
   include:
     - stage: Build + Test
       env:
+        - XCODE_WORKSPACE=Alicerce
```
Comment (portellaa) [inline .travis.yml]: huuummm not sure, but don't you have to use `./Alicerce.xcworkspace` ? 🤔

Comment (renatorodrigues) [reply .travis.yml]: I do. I gave up yesterday after so many little fixes to make Travis happy. Fixing this now.

**Comment by renatorodrigues** [comment]
Green build! 🎉🎉🎉

**Comment by renatorodrigues** [comment]
Last required addition: update the README. On it!

File: README.md [hunk 2]
```diff
@@ -92,11 +92,29 @@ If you use Swift Package Manager, simply add Alicerce as a dependency of your pa
...
+4. Build `Alicerce` scheme
+
 ## Contributing 🙌
 
-See [CONTRIBUTING](https://github.com/Mindera/Alicerce/blob/master/CONTRIBUTING.md).
+See [CONTRIBUTING].
+
+## License
+
+Alicerce is Copyright (c) 2016 Mindera and is available under the MIT License. It is free software, and may be redistributed under the terms specified in the [LICENSE] file.
```
Comment (p4checo) [inline README.md]: Shouldn't the year be 2018, or a range? (2016-2018)? 💄

Comment (renatorodrigues) [reply README.md]: Updated it to be a range. ✨

**Pull request merged**